bigbluebutton-html5/imports/startup/client/auth.js

```diff
@@ -15,6 +16,9 @@ export function joinRouteHandler(nextState, replace, callback) {
     replace({ pathname: '/error/404' });
     callback();
   }
+  
+  // Old credentials stored in memory were being used when browsers was refreshed
```

Comment:
I'm pretty sure this is expected. Are you sure you should clear credentials at this point?

Reply:
The comment should be, "Old credentials stored in memory were being used when joining a new meeting". This auth.js code is called when a user goes through the /join endpoint and clearing the old state here enforces that if there's a problem grabbing the new information that the client won't try to use the old information.

Reply:
@capilkey +1